publisher/product/ontology/build-theallfile.sh

```diff
@@ -41,8 +41,8 @@ function ontologyCreateTheAllTtlFile() {
     python3 ${SCRIPT_DIR}/lib/trigify.py \
       --dir="${INPUT}/LCC" \
       --dir=${tag_root} \
-      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/AboutFIBODev/" \
-      --top="https://spec.edmcouncil.org/fibo/ontology/MetadataFIBO/" \
+      --top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY}Dev/" \
```

Comment:
Should be uppercase:
`--top="https://spec.edmcouncil.org/${family_product_branch_tag:?}/About${ONTPUB_FAMILY^^}Dev/" \`